editor/hooks/autocomplete/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { compose } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { Autocomplete, withContext } from '@wordpress/components';
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import { blockAutocompleter, userAutocompleter } from './autocompleters';
+import { getInserterItems } from '../../store/selectors';
+
+function withAutocomplete( BlockEdit ) {
+	function WrappedBlockEdit( props ) {
+		const { name, items, onReplace } = props;
+
+		// Only wrap paragraph blocks with <Autocomplete>
+		if ( name !== 'core/paragraph' ) {
+			return <BlockEdit { ...props } />;
+		}
+
+		return (
+			<Autocomplete completers={ [ blockAutocompleter( { items, onReplace } ), userAutocompleter() ] }>
+				{ ( { isExpanded, listBoxId, activeId } ) => (
+					// TODO: These aria attributes probably belong on the <Editable>... not sure how to make that happen
```

Comment:
This is the only problem I ran into. Not sure where these aria attributes which were previously on the `<Editable>` should go.

Reply:
Yes, I agree that they fit better here than as part of `<Editable />`. I think @afercia might be better suited to answer this question.